src/web/components/Card/components/AvatarContainer.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { css } from 'emotion';
+
+type Props = {
+    children: JSXElements;
+};
+
+export const AvatarContainer = ({ children }: Props) => {
+    return (
+        <div
+            className={css`
+                display: flex;
+                flex-direction: row-reverse;
+
+                margin-right: 10px;
+                margin-top: 50px;
```

Comment:
Is this magic? Not sure what the 50px relates to exactly

Reply:
Somewhat magic. It's the space that lives above an avatar when it's inside a card